CONTRIBUTING.md

```diff
@@ -1,32 +1,228 @@
-# Contributing
+# What The Hack - Contribution Guide
 
-The What The Hack project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit <https://cla.microsoft.com>.
+What The Hack is all about being "for the people, by the people". This repo was originally created to share real-world hackathons that Microsoft employees have hosted with their customers. Here are our core principles:
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+- Anyone can use the content to [host their own WTH event](./000-HowToHack/WTH-HowToHostAHack.md).
+- Anyone can [contribute a new hack](#-contribute-a-new-hack-to-what-the-hack).
+- Anyone can modify or update a hack as needed.
+  - Submitting a [pull request for updated content](#-contribute-an-update-to-an-existing-hack) is encouraged.
+- The content can always be shared with hack attendees **(AFTER your event)**
 
-# Code of Conduct
+This document provides the guidelines for how to contribute to the What The Hack project.
```

Comment:
Recommend adding "general" in front of guidelines and make the WTH project at the end plural to projects, unless intention is to refer to the overall WTH "project"?  

Would also add statement along the lines of "If these guidelines do not cover your questions, please contact the WTH team @ ..."

Reply:
Done.  Added "general".  But intention is to refer to the overall WTH project as one thing.